nym-api/src/support/cli/run.rs

```diff
@@ -62,24 +62,29 @@ pub(crate) async fn execute(args: Args) -> anyhow::Result<()> {
 
     config.validate()?;
 
-    let mut axum_shutdown = start_nym_api_tasks_v2(&config).await?;
-    let mut shutdown_handlers = start_nym_api_tasks(config).await?;
+    #[cfg(feature = "axum")]
```

Comment:
shouldn't it be the case that if we're using `axum` feature, then the rocket server does **not** startup? I think we just want one or the other, no?

Reply:
We might need both of them (as they are now) for testing purposes, to compare whether `axum` does the same ting as `rocket `